.pre-commit-hooks.yaml

```diff
@@ -191,7 +191,7 @@
   language: system
   types:
     - ts
-  args: [--findRelatedTests]
+  args: [--passWithNoTests, --findRelatedTests]
```

Comment:
What is the purpose of keeping --findrelatedtests if the command before says --passwithnotests?

Reply:
[`--findRelatedTests`](https://jestjs.io/docs/cli#--findrelatedtests-spaceseparatedlistofsourcefiles) optimizes Jest when run as a pre-commit hook by skipping any tests that wouldn’t be expected to have any connection to any of the modified files. [`--passWithNoTests`](https://jestjs.io/docs/cli#--passwithnotests) prevents the pre-commit hook from failing when there are no tests to run. In conjunction, this makes it possible to modify files that don’t have any related tests (e.g., `jest.config.ts`) without the hook failing.